Weather/ForecastIO/weather.15m.py

```diff
@@ -201,6 +207,10 @@ def render_wx():
 
   print '---'
 
+  if 'week' in weather_data:
+    print "\n".join(textwrap.wrap(weather_data['week'], 50))
+    print '---'
+
   if 'next_hour' in weather_data:
```

Comment:
Like this?

![image](https://cloud.githubusercontent.com/assets/3022496/17386171/9e99633e-59ad-11e6-8820-a17f7ba38289.png)